server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -151,6 +151,8 @@ public static ThreadPoolType fromType(String type) {
         entry(Names.SYSTEM_CRITICAL_WRITE, ThreadPoolType.FIXED)
     );
 
+    public static final double searchAutoscalingEWMA = 0.1;
```

Comment:
See https://github.com/elastic/elasticsearch-serverless/pull/445 for background on this number.